sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -353,8 +417,12 @@ def _transmit(self, envelopes: List[TelemetryItem]) -> ExportResult:
                             from azure.monitor.opentelemetry.exporter.statsbeat._statsbeat import (
                                 shutdown_statsbeat_metrics,
                             )
+                            from azure.monitor.opentelemetry.exporter.statsbeat._customer_statsbeat import (
+                                shutdown_customer_statsbeat_metrics,
+                            )
 
                             shutdown_statsbeat_metrics()
+                            shutdown_customer_statsbeat_metrics()
```

Comment:
We actually did not spec this shutdown logic yet for customer statsbeat. Also this is dependent on the `_is_statsbeat_initializing_state` and `reach_ingestion` which is only supported for regular statsbeat so it doesn't make sense for it to be here.